lib/clowne/adapters/active_record/associations/base.rb

```diff
@@ -48,6 +48,10 @@ def with_scope
 
           private
 
+          def clone_record(record)
+            Clowne.resolve_adapter(:active_record).copier.call(record)
```

Comment:
Frankly speaking, I don't think it make sense to separate _pure_ cloning logic from adapter.
If you need custom _copier_ then just add another adapter (e.g. `:custom_active_record`).

Reply:
Or even better: add specific declaration (`initialize_with ->(source) { ... }`) to the configuration to make it possible to re-define default adapter behaviour in-place.

Anyway, let's postpone this change.

Reply:
Answered in a email ;)

Reply:
@palkan https://github.com/palkan/clowne/pull/4/commits/656089891a45bbc2318a5a1fa66b1cd2af6dbe9c